openedx/features/caliper_tracking/tests/expected/edx.course.enrollment.deactivated.json

```diff
@@ -0,0 +1,33 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "id": "urn:uuid:fa25a902-275e-4e5b-90b6-bd2779608dce",
+  "type": "Event",
+  "actor": {
+    "id": "edx.devstack.lms:18000/u/honor",
+    "type": "Person",
+    "name": "honor"
+  },
+  "action": "Deactivated",
+  "object": {
+    "id": "http://localhost:18000/courses/course-v1:edx+cs-101+2018",
+    "type": "CourseOffering"
+  },
+  "referrer": "http://localhost:18000/dashboard",
+  "eventTime": "2018-10-17T12:14:54.844Z",
+  "extensions": {
+    "extra_fields": {
+      "accept_language": "en-US,en;q=0.9,ur-PK;q=0.8,ur;q=0.7",
+      "agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36",
```

Comment:
Please use same value for `agent` in both files.

Reply:
Fixed.